bin/index.js

```diff
@@ -86,6 +96,12 @@ module.exports = function(Aquifer, AquiferGitConfig) {
       return false;
     }
 
+    // Validate custom signature options.
+    if (!options.name != !options.email) {
```

Comment:
Triple negatives are there to replicate XOR. The negatives on either side of the operator ensures that both sides are boolean values. Is there a better way to do this that's not too verbose?